python.cmake

```diff
@@ -77,25 +77,20 @@ GET_FILENAME_COMPONENT(PYTHON_LIBRARY_DIRS "${PYTHON_LIBRARIES}" PATH)
 MESSAGE(STATUS "PythonLibraryDirs: ${PYTHON_LIBRARY_DIRS}")
 MESSAGE(STATUS "PythonLibVersionString: ${PYTHONLIBS_VERSION_STRING}")
 
-# Default Python packages directory
-SET(PYTHON_PACKAGES_DIR site-packages)
-
 # Use either site-packages (default) or dist-packages (Debian packages) directory
 OPTION(PYTHON_DEB_LAYOUT "Enable Debian-style Python package layout" OFF)
 
-IF (PYTHON_DEB_LAYOUT)
-  SET(PYTHON_PACKAGES_DIR dist-packages)
-ENDIF (PYTHON_DEB_LAYOUT)
-
 EXECUTE_PROCESS(
   COMMAND "${PYTHON_EXECUTABLE}" "-c"
-  "import sys, os; print(os.sep.join(['lib', 'python' + sys.version[:3], '${PYTHON_PACKAGES_DIR}']))"
+  "from distutils import sysconfig; print(sysconfig.get_python_lib(prefix='', plat_specific=True))"
```

Comment:
`plat_specific=True` means e.g. `lib64` vs `lib` (`site-packages` vs `dist-packages` is OS-specific). Is that intended and do you need that platform-specific for your use cases - it will change existing lib to lib64 etc.? [just tested on Ubuntu 18.04 and RHEL]

Reply:
You're right. I will change it. Thanks for this quick feedback.

Reply:
Done!

Reply:
Sorry, I am not seeing the change reflected here - did you already push?

Reply:
Sorry, I indeed forgot to push. Thanks :)